api/controllers/accounts.js

```diff
@@ -65,7 +65,10 @@ function accountFormatter(totalSupply, account) {
 		object.delegate.rank = parseInt(object.delegate.rank);
 
 		// Computed fields
-		object.delegate.approval = calculateApproval(object.vote, totalSupply);
+		object.delegate.approval = calculateApproval(
```

Comment:
use of variable `object` is not ok here, `Object` is a reserved keyword and also the variable can be named better here, since its related to `account` can we name it better? like `formatAccount` or something more meaningful.

Reply:
@lsilvs why is it suddenly split into 4 lines:

from:
```
object.delegate.approval = calculateApproval(object.vote, totalSupply);
```
to
```
object.delegate.approval = calculateApproval(
			object.delegate.vote,
			totalSupply
		);
```

@mitsujutsu in his PRs has a similar behaviour

Reply:
@ManuGowda I agree with your comment but your request is out of scope of this PR.

Reply:
@MaciejBaj I believe it was split because adding `delegate` property increased the length of the line requiring this format.

Reply:
@LucasIsasmendi can you create an issue to at least fix it in the later stage, I agree it's not in the scope, but its something we need to fix later.

Reply:
@ManuGowda  Can you open the issue as it's your request?

You tagged the wrong Lucas haha.

Reply:
😄 ok will create one.